client/src/components/AddNewProfile.js

```diff
@@ -173,6 +221,22 @@ const AddNewProfile = () => {
           options={statusOptions}
         />
       </Form.Field>
+      <Form.Field>
+        <label>Nationality</label>
+      <Dropdown
+            // button
+            // labeled
+            // icon="world"
+            // className="icon"
+            name="nationality"
+            value={profileData.nationality}
+            fluid
+            selection
+            options={nationalityOptions}
+            onChange={handleDropdownAndDateChange}
+            placeholder="Nationality"
+          />
+</Form.Field>
```

Comment:
Make sure your code is indented properly (try turning on the "format on save" setting in vscode if it is off).